js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-design.html

```diff
@@ -0,0 +1,52 @@
+<div data-bind="ifnot: $component.isViewPermitted(), css: `${$component.isViewPermitted() ? '' : 'flexed'}`">
+    <access-denied params="isAuthenticated: $component.isAuthenticated" class="flexed"></access-denied>
```

Comment:
I guess this will be handled by `AuthorizedRoute`

Reply:
Form where does the `AuthorizedRoute` know about CC specific-permissions?

Reply:
The page itself won't be shown until user authenticates

Reply:
How is the `AuthroizedRoute` going to know about `$component.isViewPermitted()`?